app/main/forms/suppliers.py

```diff
@@ -22,28 +22,22 @@ class EditSupplierForm(Form):
     description = StripWhitespaceStringField('Supplier summary', validators=[
         word_length(50, 'Your summary must not be more than %d words')
     ])
-    clients = FieldList(StripWhitespaceStringField())
-
-    def validate_clients(form, field):
-        if len(field.data) > 10:
-            raise ValidationError('You must have 10 or fewer clients')
 
 
 class EditContactInformationForm(Form):
     id = IntegerField()
-    address1 = StripWhitespaceStringField('Registered office address')
-    city = StripWhitespaceStringField('Town or city')
-    postcode = StripWhitespaceStringField()
-    website = StripWhitespaceStringField()
-    phoneNumber = StripWhitespaceStringField('Phone number')
-    email = StripWhitespaceStringField('Email address', validators=[
+    contactName = StripWhitespaceStringField('Contact name', validators=[
+        DataRequired(message="You must provide a contact name"),
+    ])
+    email = StripWhitespaceStringField('Contact email', validators=[
         DataRequired(message="You must provide an email address"),
         Regexp("^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$",
```

Comment:
This is the regex that turns out to be wrong but I'm fixing it in pulling out a specific `EmailField`

Reply:
The only thing changed in this PR is the order of fields and the words ("Email address" -> "Contact email"), so this could go in without the updated regex and functionality wouldn't change from what we have at the moment.

Reply:
Absolutely yes just pointing it out